IPs/RSKIP141.md

```diff
@@ -0,0 +1,59 @@
+---
+rskip: 141
+title: "Network upgrade meta: Wasabi+1"
+author: AE, SDL, JR, DLL, MM, JD, PP, JP
+type: Meta
+status: Draft
+created: 2019-09-27
+requires: Wasabi
+---
+
+## Abstract
+
+This meta-RSKIP specifies the changes included in the Ethereum hardfork name TBD.
+
+## Specification
+
+- Codename: TBD
+- Activation: TBD
+
+### Included RSKIPs
+
+- TBD
+
+### Accepted RSKIPs
+
+- Add EXTCODEHASH opcode
+- Add CHAINID opcode (ref. [EIP-1344](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1344.md))
+- 2WP Locking Cap
+- 2WP Onchain Events
+- ValidateEthPow precompiled contract
+
+### Tentatively Accepted EIPs
+
+- ECADD, ECMUL and Pairing check precompiled contract
+- SELF_BALANCE opcode and SLOAD, BALANCE and EXTCODEHASH cost reduction (ref. [EIP-1884](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1884.md))
```

Comment:
Most are cost increases rather than cost reductions. only SELF_BALANCE is a reduction.